lomap/graphgen.py

```diff
@@ -166,6 +172,31 @@ def __init__(self, dbase):
 
         return
 
+    #sliu 07/17 add function to pick the lead compounds for the radial graphing option
+    def pick_lead(self):
+        if (self.dbase.nums() * (self.dbase.nums() - 1)/2) != self.dbase.strict_mtx.size:
```

Comment:
Can you add a comment saying how you know there is a problem if this condition is met? It's not immediately obvious to me.